test/consts.go

```diff
@@ -11,5 +11,5 @@ const (
 	// Running on Pod tcp-echo-58ccd6b78d-hn9t8.
 	// In namespace foo.
 	// With IP address 10.244.0.13.
-	TCPEchoImage = "kong/go-echo:0.2.0"
+	TCPEchoImage = "kong/go-echo:0.3.0"
```

Comment:
Since this is used in e2e tests : https://github.com/Kong/kubernetes-ingress-controller/blob/52aea3ea31dd1d02b3e86e5b59b0afab8ad51bb7/test/e2e/features_test.go#L440 
I'd trigger those before merging to be sure it works.